erts/preloaded/src/erlang.erl

```diff
@@ -1642,7 +1644,7 @@ put(_Key, _Val) ->
     erlang:nif_error(undefined).
 
 %% raise/3
--spec erlang:raise(Class, Reason, Stacktrace) -> no_return() when
+-spec erlang:raise(Class, Reason, Stacktrace) -> badarg when
```

Comment:
Note that 'badarg' is an atom.

Reply:
Yes, but in a type spec `badarg` reffers to the type of the atom badarg, or have I done something wrong? Btw, is it better to write the return type as `no_return() | 'badarg'` in this case to illustrate that the function may throw an exception?

Reply:
My bad. Too fast on the trigger. No, 'badarg' is OK, of course.

Reply:
It is recommended to single-quote atoms in specs.

Reply:
Fixed in 0d1d237